doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -149,31 +132,40 @@ Programmable proof search
    .. seealso:: :ref:`The Hints Databases for auto and eauto <thehintsdatabasesforautoandeauto>`
 
 
-.. tacn:: autounfold with {+ @ident}
-   :name: autounfold
+.. tacn:: autounfold {? @hintbases } {? @occurrences }
 
-   This tactic unfolds constants that were declared through a :cmd:`Hint Unfold`
+   Unfolds constants that were declared through a :cmd:`Hint Unfold`
    in the given databases.
 
-.. tacv:: autounfold with {+ @ident} in @goal_occurrences
+   :n:`in @occurrences`
```

Comment:
`@occurrences` already contains the `in`. You should just write down that only the `in @goal_occurrences` case of `@occurrences` is supported.

Reply:
Not quite that simple--`at` is not supported inside of `in @goal_occurrences`, either.